compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -165,6 +165,25 @@ pub struct ImplHeader<'tcx> {
     pub predicates: Vec<Predicate<'tcx>>,
 }
 
+#[derive(
```

Comment:
This would probably look better as two separate derive lines...

Reply:
I was just following what rustfmt/existing code was doing.